common/db.go

```diff
@@ -217,6 +217,11 @@ func InitDB(dbPath string) (*DB, error) {
 		ELSE excluded.deactivationRound END 
 	`)
 	d.updateOrch = stmt
+	if err != nil {
+		glog.Error("unable to prepare updateOrch ", err)
```

Comment:
Capitalize "unable" since all the other error messages used when preparing statements capitalize it?

Reply:
fixed